scripts/rewrite/create-issues.py

```
@@ -0,0 +1,359 @@
+                flush_key()
+                continue
+            buffer.append(stripped)
+
+    flush_key()
+    if current is not None:
+        tickets.append(current)
+    return tickets
+
+
+def gh(*args: str, check: bool = True, capture: bool = True) -> subprocess.CompletedProcess:
+    return subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)